### ISSUE TITLE
Add daily, monthly entity in Chinese in NER

### ISSUE THREAD

**Issue by kilik52** [axa-group/nlp.js#223]
Consider this phrase in English:

**'monthly sales in 2019'**

the word **monthly** will be recognized by NER manager and have the following value:

````json
{
	"start": 0,
	"end": 6,
	"len": 7,
	"accuracy": 0.95,
	"sourceText": "monthly",
	"utteranceText": "monthly",
	"entity": "set",
	"resolution": {
		"values": [{
			"timex": "P1M",
			"type": "set",
			"value": "not resolved"
		}]
	}
}
````

(By the way, the entity type **set** can not be found in the document.)

In Chinese, the word which represents monthly is **月**, and the tricky part is, month names like January, February writes in Chinese like: 1月, 2月. So if i add a named entity **月** to represent **monthly**, it will conflict with data range types: 1月, 2月.

In conclusion, i'd like to achieve this:

1月 -> recognized as datarange, January.  

月 -> recognized as set, monthly.

The example phrase is like:

2019年月销售额 -> monthly sales in 2019

1月日销售额 -> daily sales in January

Is there a way to achieve this? thank you.

**Comment by jesus-seijas-sp** [comment]
The builtin entity recognition is done using the Recognizers-Text implemented by Microsoft that you can find here: https://github.com/microsoft/Recognizers-Text

I'm thinking about doing my own implementation due to the fact that their cycles to publish are too long (9 months since the last version!!!!)

**Comment by kilik52** [comment]
> The builtin entity recognition is done using the Recognizers-Text implemented by Microsoft that you can find here: https://github.com/microsoft/Recognizers-Text
> 
> I'm thinking about doing my own implementation due to the fact that their cycles to publish are too long (9 months since the last version!!!!)

ok, thanks for the reply
